src/Pages/AboutUs/AboutUs.css

```diff
@@ -0,0 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,700;1,300&&display=swap');
+
+section {
```

Comment:
we should scope the css, meaning like you have done below, include a classname with the `section` tag.

when we change the css for a `section` here, react applies it everywhere and we may not want the same css in other places